src/Constants/Paths.cs

```diff
@@ -0,0 +1,28 @@
+namespace fiskaltrust.Launcher.Constants
+{
+    public static class Paths
+    {
+        public static string ServiceFolder
+        {
+            get
+            {
+                if (OperatingSystem.IsWindows())
+                {
+                    return "C:/ProgramData/fiskaltrust";
+                }
+                else if (OperatingSystem.IsLinux())
```

Comment:
Both of them need root permissions, I think

Reply:
Probably okay, but we should document that :D

Reply:
You're right, but I think you "only" need root access during installation which is common for `apt` and `homebrew`